### ISSUE TITLE
Future warning with import_pbp_data()

### ISSUE THREAD

**Issue by raerickson** [cooperdff/nfl_data_py#42]
Great package. Thank you for your hard work!

I searched through the repo and didn't find any issues for this issue. I'm using Python version 3.10 and Pandas version 1.5.2 and obtain a future warning when running `import_pbp_data()`:

```python
import nfl_data_py as nfl
pbp_py = nfl.import_pbp_data([2022])
```

```python
/home/raerickson/miniconda3/envs/football/lib/python3.10/site-packages/nfl_data_py/__init__.py:137: FutureWarning: In a future version, `df.iloc[:, i] = newvals` will attempt to set the values inplace instead of always setting a new array. To retain the old behavior, use either `df[df.columns[i]] = newvals` or, if columns are non-unique, `df.isetitem(i, newvals)`
  plays.loc[:, cols] = plays.loc[:, cols].astype(numpy.float32)
```
Sorry, but I don't have time to dig in, but I think this is a quick fix.

**Comment by alecglen** [comment]
Hey @raerickson, appreciate the report! It seems that is actually a Pandas issue they're working on currently. https://github.com/pandas-dev/pandas/issues/48673

**Comment by raerickson** [comment]
@alecglen Thanks for digging in. Should my issue be closed? Or do you want to leave it open in case others try to report the bug.

**Comment by alecglen** [comment]
@raerickson This can be closed now. The issue should be resolved if you update your Pandas to version 1.5.3+.
